cachito/common/packages_data.py

```diff
@@ -6,7 +6,7 @@
 from pathlib import Path
 from typing import Any, Dict, Iterator, List, Tuple, Union
 
-from cachito.errors import CachitoError
+from cachito.common.errors import CachitoError
```

Comment:
Maybe `errors` could stay in `cachito.errors`, it's pretty common to keep exceptions in `<project>.errors`